web/src/components/alerts/ErrorPage.tsx

```diff
@@ -17,11 +17,19 @@ export const ErrorPage = ({ error, resetErrorBoundary }: FallbackProps) => {
     }
   });
 
-  return (
+  var online = true;
+  const backendEndpoint = window.location.host + "/api/healthz/liveness"
+  fetch(backendEndpoint, { method: 'HEAD' })
+  .catch(() => {
+    online = false
+  });
```

Comment:
Would recommend to add this in the APIClient sicne that handles paths with baseUrl.